osu.Game/Rulesets/Edit/BeatmapVerifierContext.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Beatmaps;
+
```

Comment:
```suggestion
 #nullable enable
```

Best to use this in non-drawable classes to make it explicit as to which properties should be nullable.